notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1294 @@
+{
```

Comment:
<p>Can we define L-inifnity distance and Jensen-Shannon Divergence in a few words? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>+1 </p><p> </p><p>I also think we need brief definitions here. </p>

Reply:
<p>Done. </p>